model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -156,7 +156,7 @@ object Workspace {
     createdBy: String,
     attributes: AttributeMap,
     isLocked: Boolean = false): Workspace = {
-    new Workspace(namespace, name, workspaceId, bucketName, workflowCollectionName, createdDate, lastModified, createdBy, attributes, isLocked, WorkspaceVersions.V1, GoogleProjectId(namespace), None)
+    new Workspace(namespace, name, workspaceId, bucketName, workflowCollectionName, createdDate, lastModified, createdBy, attributes, isLocked, WorkspaceVersions.V2, GoogleProjectId(namespace), None)
```

Comment:
I wouldn't want to change the actual constructor in the code to default the tests. I would rather have our test setup code start using the new constructor with more details. I think this has a high potential to give us issues setting up our migration strategy, since WorkspaceVersions was how we were going to detect workspaces that haven't been migrated (I believe?)

Reply:
It seems like this would be more involved than we initially thought. I can't move this object to a test file because it is a companion object and companion objects are "declared in the same file as a class" (https://docs.scala-lang.org/overviews/scala-book/companion-objects.html)